fido_metadata.gemspec

```diff
@@ -35,6 +35,6 @@ Gem::Specification.new do |spec|
   spec.add_dependency "jwt", "~> 2.0"
   spec.add_development_dependency "rake", "~> 13.0"
   spec.add_development_dependency "rspec", "~> 3.8"
-  spec.add_development_dependency "rubocop", "0.75.0"
+  spec.add_development_dependency "rubocop", "~> 0.81"
```

Comment:
Perhaps we could only install `rubocop` if on the latest Ruby version? That way we can always install the latest Rubocop version and the `TargetRubyVersion` config from Rubocop would ensure that it knows that the code can be run in Ruby 2.3